backend/app/models.py

```diff
@@ -1,43 +1,40 @@
+from pydantic import EmailStr
 from sqlmodel import Field, Relationship, SQLModel
 
 
 # Shared properties
-# TODO replace email str with EmailStr when sqlmodel supports it
 class UserBase(SQLModel):
-    email: str = Field(unique=True, index=True)
+    email: EmailStr = Field(unique=True, index=True, max_length=255)
     is_active: bool = True
     is_superuser: bool = False
-    full_name: str | None = None
+    full_name: str | None = Field(default=None, max_length=255)
 
 
 # Properties to receive via API on creation
 class UserCreate(UserBase):
-    password: str
+    password: str = Field(min_length=8, max_length=40)
```

Comment:
Could you please update the min length of passwords to the same of the other text fields, with 255?

Reply:
I think it should be the max lenght?

Reply:
Yep, right @PhilippWu !

Reply:
Chatting internally, @estebanx64 convinced me to leave this with 40. :sweat_smile:

Reply:
Hi @tiangolo, would you be able to share the logic on the 40? Sounds counterintuitive to me, so something to learn.